node/tests/string.go

```diff
@@ -0,0 +1,71 @@
+package tests
+
+import (
+	ipld "github.com/ipld/go-ipld-prime"
+	"github.com/ipld/go-ipld-prime/node/mixins"
+)
+
+// A trimmed down copy from node/basic,
+// for the sake of test fixtures without import cycles.
+// TODO(mvdan): consider, instead, making node/basic tests be external
```

Comment:
I think this would work, but I also want to avoid going down rabbit holes before I can "publish" a v0.1 of basicnode.

Reply:
If this is something that can be fixed by painting a `_test` suffix onto some package declarations somewhere, yeah, go for it.

Reply:
it's a bit more than that, as I also have to rewrite a bunch of expressions to use the package qualifier. I'll give it a quick go.

Reply:
done